### DISCUSSION TITLE
quploader get reference to files

### DISCUSSION THREAD

**Discussion by bitmandk** [quasarframework/quasar#15333] [General - Components / Directives / etc] [answered]
How can I get a reference (in script) to the files in Quploader, more specifically after upload (or failure) I want to remove the files that was sucesfully uploaded and keep the failed files in the list. So my idea was to use @uploaded and iterate through the files and remove the files with status "success" and keep the files with status "failed", but when using `this.$refs.theUploader.files` I just get an empty array, shown here in the codePen (using onFailure): 

https://codepen.io/bobittjek/pen/QWBxvoj?editors=1111

**Comment by yusufkandemir** [comment, marked as answer]
It's a bug, see #14967
